eva_submission/eload_submission.py

```diff
@@ -133,8 +133,8 @@ def update_metadata_from_config(self, input_spreadsheet, output_spreadsheet=None
                 })
 
         project_row = reader.project
-        if self.eload_cfg.query('brokering', 'ena', 'existing_project'):
-            project_row['Project Alias'] = self.eload_cfg.query('brokering', 'ena', 'PROJECT')
+        if existing_project:
+            project_row['Project Alias'] = existing_project
```

Comment:
I guess this technically isn't updating metadata from just the config anymore, as the method name states... it might get confusing if we keep doing this, but it does make sense in this case...